### PR TITLE
Activation Key documentation update

### PR DESCRIPTION
**Pull request by sseelam2**

### CODE REVIEW

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 1]
```diff
@@ -13,12 +13,14 @@ Activation Keys provide a mechanism to define properties that may be applied to
...
 - Repository Enablement
 - Host Collections
+- System Purpose
 
 ## Definitions
 
 - [Content Host](/plugins/katello/{{ page.version }}/user_guide/content_hosts/index.html#what-are-content-hosts)
 - Host Collection - A statically defined group of Content Hosts.
 - Subscription - The right to receive the associated content from Katello.
+- System Purpose - When a system’s purpose is set, the entitlement server receives information that helps auto-attach a subscription that satisfies the intended use of the system.
```
Comment (jturel) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: Let's phrase this: `Helps auto-attach find a subscription that satisfies the intended use of the system`

**Review by jturel — CHANGES_REQUESTED** [review]
Looks great. The screenshots on my main monitor look fine - I think we can keep them.

Left one comment on the description for System Purpose.

Can you give Content Hosts the same treatment since they also relate to System Purpose and have outdated screenshots?

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 2]
```diff
@@ -86,24 +90,19 @@ Finally, when 'Auto-Attach' is disabled, all subscriptions on the activation key
...
 To change repository enablement settings using a key:
 
 - navigate to: Content > Activation Keys
 - select the desired key from the list
-- click **Product Content**
-- click the edit icon for the repository content set that you would like to modify
-- select the desired value (e.g. 'Override to Yes', 'Override to No', 'Defaults to Yes', 'Defaults to No')
-- click **Save**
+- click **Repository Sets**
+- select the repository that you would like to modify
```
Comment (swadeley) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: s/select the repository that you would like to modify/select the repository you want to modify/  
{less words, easier to translate}

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 3]
```diff
@@ -86,24 +90,19 @@ Finally, when 'Auto-Attach' is disabled, all subscriptions on the activation key
...
+- select the desired value under the **Select Action** drop down(e.g. 'Override to Enabled', 'Override to Disable', 'Reset to Default')
```
Comment (swadeley) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: Avoid "desire" in technical writing: s/select the desired value under the **Select Action** drop down(e.g. 'Override to Enabled', 'Override to Disable', 'Reset to Default')/select the value you want to change from the **Select Action** menu ('Override to Enabled', 'Override to Disable', or 'Reset to Default')/  {no need to use e.g. when you are giving a complete list}

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 4]
```diff
@@ -118,6 +117,26 @@ To add Host Collections to a key:
 
 ![Adding Host Collections to an Activation Key](/plugins/katello/{{ page.version }}/user_guide/activation_keys/activation_key_add_host_collections.png)
 
+## Add System Purpose details to an Activation Key
+
+To add System Purpose to a key:
+
+- navigate to: Content > Activation Keys
+- select the desired key from the list
```
Comment (swadeley) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: s/select the desired key from the list/select the key you want from the list/

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 5]
```diff
@@ -118,6 +117,26 @@ To add Host Collections to a key:
...
+- There is a section for System Purpose where you can set the Service Level, Usage Type, Role and Add-ons for a system.
```
Comment (swadeley) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: s/ Role and Add-ons/ Role, and Add-ons/ Oxford comma if Role and Add-ons are not pair but separate items in the list

File: plugins/katello/nightly/user_guide/activation_keys/index.md [hunk 6]
```diff
@@ -118,6 +117,26 @@ To add Host Collections to a key:
...
+- There is a section for System Purpose where you can set the Service Level, Usage Type, Role and Add-ons for a system. 
+
+![Adding System Purpose to an Activation Key](/plugins/katello/{{ page.version }}/user_guide/activation_keys/activation_key_system_purpose.png)
+
+## View Content Hosts registered with the Activation Key
+
+To view Content Hosts registered with a particular Activation Key:
+
+- navigate to: Content > Activation Keys
+- select the desired key from the list
```
Comment (swadeley) [inline plugins/katello/nightly/user_guide/activation_keys/index.md]: s/select the desired key from the list/select the key you want from the list/

**Comment by sseelam2** [comment]
@jturel @swadeley Thank you for your feedback. I made the changes for activation key. Working on content hosts documentation.

File: plugins/katello/nightly/user_guide/activation_keys/index.md (hunk 1 above)
Comment (jturel) [reply plugins/katello/nightly/user_guide/activation_keys/index.md]: Looks like you missed this in your recent update

Comment (sseelam2) [reply plugins/katello/nightly/user_guide/activation_keys/index.md]: Ah. You meant change the whole definition of System Purpose. My bad. I just changed the last part of the sentence.

**Comment by sseelam2** [comment]
> Can you give Content Hosts the same treatment since they also relate to System Purpose and have outdated screenshots?

@jturel I have edited the Content Host documentation to include information about System Purpose. Let me know if anything needs to be changed.

**Review by jturel — APPROVED** [review]
Nice work. Thanks Swetha!

**Review by jturel — CHANGES_REQUESTED** [review]
I see that the Katello 3.13 docs have been merged in which means that the changes here need to be in nightly (which you have) and also copied into the 3.13 folder since this was work completed for 3.13

**Comment by sseelam2** [comment]
> I see that the Katello 3.13 docs have been merged in which means that the changes here need to be in nightly (which you have) and also copied into the 3.13 folder since this was work completed for 3.13

@jturel I have pushed the branch with requested changes.

File: plugins/katello/3.13/user_guide/activation_keys/index.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugins/katello/documentation
 title: Activation Keys
-version: 3.13
+version: nightly
```
Comment (jturel) [inline plugins/katello/3.13/user_guide/activation_keys/index.md]: version should be 3.13 here

File: plugins/katello/3.13/user_guide/content_hosts/index.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugins/katello/documentation
 title: Content Hosts
-version: 3.13
+version: nightly
```
Comment (jturel) [inline plugins/katello/3.13/user_guide/content_hosts/index.md]: version should be 3.13 here

**Review by jturel — CHANGES_REQUESTED** [review]
2 small notes!

File: plugins/katello/3.13/user_guide/content_hosts/index.md (hunk 8 above)
Comment (sseelam2) [reply plugins/katello/3.13/user_guide/content_hosts/index.md]: Thanks for noticing that :+1: @jturel

**Review by jturel — APPROVED** [review]
ACK!

Reach out to theforeman-dev for a merge - I don't have access here

**Pull request merged**